File: University/AdadChopKonS1.php
<?php

class Main {
    function __construct(){
    }
    public static function Main_1() {
        $local_this = new Main();
        return $local_this;
    }
    public static function main_1(&$args) {
        try (
        public $scanner = "Inputs";)
        {
            $string = readline();
            for ($i = 0; $i < strlen($string); $i++)
            {
                echo strval($string[$i]) . ": ";
                for ($j = 0; $j < ord($string[$i]) - 48; $j++)
                {

echo $string[$i];
                }
                echo "","\n";
            }
        }
    }
}
Main::main_1($argv);

File: University/AdadChopKonS1.cs
using System;
using System.IO;
public class Main
{
    public static void Main(string[] args)
    {
        try (public const  scanner =  "Inputs";)
        {
            var string = Console.ReadLine();
            for (int i = 0; i < string.Length; i++)
            {
                Console.Write(string[i].ToString() + ": ");
                for (int j = 0; j < (int)(string[i]) - 48; j++)
                {
                    Console.Write(string[i]);
                }
                Console.WriteLine("");
            }
        }
    }
}
